static/js/stream_edit.js

```diff
@@ -298,32 +326,16 @@ function show_subscription_settings(sub_row) {
     const users = exports.get_users_from_subscribers(sub.subscribers);
     exports.sort_but_pin_current_user_on_top(users);
 
+    const list_container = get_subscriber_list_container(sub_settings);
+    list_container.ready(() => {
+        render_subscribers_list(stream_id, list, users);
+    });
```

Comment:
https://api.jquery.com/ready/ suggests this is a deprecated way to do things.  I'm not sure I understand what the `ready` here is doing -- is it just adding a bit of delay?  Would `setTimeout(, 0)` do the same thing?  

I don't really understand why this chang change would help -- we're not checking for something being visible, so it shouldn't matter if the DOM has a bit of time to wait.

More investigating is needed.

Reply:
I actually made this change so that, the list is created only after simplebar is added to this element and the simplebar code says that the simple is created only when document is loaded into the DOM. 
But the fix here 404604a by aman is better approach than this which just create the simplebar if not done already rather than waiting.

Reply:
OK, closing in favor of clearer approach.